spec/statsd/sender_spec.rb

```diff
@@ -48,6 +48,13 @@
           |thds| thds.any? { |t| t.name == "Statsd Sender" }
         }
       end
+
+      it 'marks the sender thread as fork safe' do
+        subject.start
+        expect(Thread.list).to satisfy {
+          |thds| thds.any? { |t| t.thread_variable_get(:fork_safe) }
```

Comment:
presumably you'd want to check the actual sender thread here, no? 

```suggestion
          |thds| thds.any? { |t| t.name == "Statsd Sender" && t.thread_variable_get(:fork_safe) }
```

Reply:
Updated 👍🏽